_posts/2022-03-07-gsoc2022.md

```diff
@@ -0,0 +1,126 @@
+check the [ideas][] page — we might add more ideas after the posting date of
+this post.
+
+You don't need to have all the skills that we are listing, although that helps,
+of course. But you need to demonstrate to us that you're able and keen to learn
+anything that you don't know yet, and we will be happy to help you learn during
+your project with us. 
+
+| project | name                                                                                                                                                      | difficulty | project size | description                                                                                    | skills                      | mentors                                |
+|---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|------------|------------|------------------------------------------------------------------------------------------------|-----------------------------|----------------------------------------|
+| 1       | [Generalise Groups]({{ site.github.wiki }}/GSoC-2022-Project-Ideas#project-1-bead-and-ring-groups)                                         | medium     | 350 hours   | Generalise concept of groups                                                                   | Python, NetworkX, Molecular modelling                  | @lilyminium, @fiona-naughton, @richardjgowers, @IAlibay, @micaela-matta |
+| 2       | [Type hinting]({{ site.github.wiki }}/GSoC-2022-Project-Ideas#project-2-type-hinting)                              | medium      | 175 hours   | Add type hints to the MDAnalysis library                                                          | Python        | @IAlibay, @jbarnoud |
+| 3       | [Extend MDAnalysis Interoperability]({{ site.github.wiki }}/GSoC-2022-Project-Ideas#project-3-extend-interoperability)                  | medium     | 350 hours   | Extend converters module to other relevant packages                                            | Python, Molecular Modelling                      | @lilyminium, @IAlibay, @fiona-naughton, @hmacdope  |
+| 4       | [Benchmarking and performance optimization]({{ site.github.wiki }}/GSoC-2022-Project-Ideas#project-4-benchmarking-and-performance-optimization)                  | medium     | 175 hours   |  write benchmarks for automated performance analysis and address performance bottlenecks | Python                      | @hmacdope, @orbeckst, @jbarnoud  |
+| 5       | [Context-aware guessers]({{ site.github.wiki }}/GSoC-2022-Project-Ideas#project-5-context-aware-guessers) | medium | 350 hours   | Extend how the library guesses properties such as bonds, masses or atom symbols; and write guessers that know about the context of the system (database of origin, force field...) | Python, Molecular modelling | @jbarnoud @micaela-matta @IAlibay |
+
+# Information for Students
```

Comment:
For here and everywhere else that says "students".

Due to the recent changes, GSoC no longer calls folks that take part solely "students" but also "contributors".

From the mailing list:

"With this change we needed a new way to address the participants who are not students so we came up with the term “GSoC Contributors”​​ as it is more welcoming than participants. We are trying to use the full term “GSoC Contributor” so they aren’t confused in a general conversation about contributors already in your community. "

It would be good for us to switch to "GSoC Contributors" as our main wording from now on?

Reply:
yes, thanks for reading carefully